.changeset/stale-lions-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore-backend': patch
+---
+
+Added support for new backend system (explore)
```

Comment:
This will be part of the Changelog so the scope of "explore" should be clear :)

```suggestion
Added support for new backend system
```

Reply:
@tudi2d Thanks, I have made the change.